src/scenes/home/home.js

```diff
@@ -194,6 +195,7 @@ class Home extends Component {
             <Route path="/terms" component={Terms} />
             <Route path="/chapter_leader" component={ChapterLeader} />
             <Route path="/leadership_circle" component={LeadershipCircle} />
+            <Route path="/get-involved" component={GetInvolved} />
```

Comment:
can you also add a Route for `"/get_involved"`. I think you can use `<Redirect>`

Reply:
@kylemh Is this due to usage of kebap-case vs snake_case? This won't be the only route that could use redirect to address that - there's several others in home.js already - we've had inconsistencies with route naming for quite some time.  

Let's just create another issue to add redirects for all of those at once, so we can finish up this Get Involved page.